app/Community/Controllers/ForumTopicController.php

```diff
@@ -91,4 +97,41 @@ public function destroy(ForumTopic $topic): void
     {
         $this->authorize('delete', $topic);
     }
+
+    public function recentlyActive(): InertiaResponse
+    {
+        $recentlyActiveTopics = ForumTopic::query()
+            ->select('ForumTopic.*', 'ftc24h.comment_count as comment_count_24h', 'ftc24h.oldest_comment_id as oldest_comment_id_24h', 'ftc7d.comment_count as comment_count_7d', 'ftc7d.oldest_comment_id as oldest_comment_id_7d')
+            ->with(['latestComment.user'])
+            ->leftJoinSub($this->buildCommentCountSubQuery('24 HOUR'), 'ftc24h', 'ForumTopic.ID', '=', 'ftc24h.ForumTopicID')
```

Comment:
The back-end code to build the page begins here. It works like this:
* Make a query to the DB.
* Map the query results to a Resource.
* Return props as paginated data from `LengthAwarePaginator`.